packages/flutter/lib/src/services/raw_keyboard.dart

```diff
@@ -499,12 +499,71 @@ class RawKeyUpEvent extends RawKeyEvent {
   }) : super(data: data, character: character);
 }
 
+/// An enum that describes how to handle a key event.
+enum KeyEventDisposition {
+  /// The key event has been handled, and the event should not be propagated to
+  /// other key event handlers.
+  handled,
+  /// The key event has not been handled, and the event should continue to be
+  /// propagated to other key event handlers, even non-Flutter ones.
+  ignored,
+  /// The key event has not been handled, but the key event should not be
+  /// propagated to other Flutter key event handlers, but should be propagated
+  /// to text fields, and non-Flutter key event handlers on the platform.
+  stopPropagation,
```

Comment:
It's confusing that stopPropagation actual means to continue propagating the event to non-Flutter key handlers... I realize that coming up with a good name here will be hard, though....

Reply:
Should we maybe call this "skip" as in "skip processing this event in the framework and hand it back to the platform embedder to do whatever it wants to do with it? Not sure if that's clearer?

Reply:
Yeah, I don't know what a better name is either (and I'm not terribly happy with it either).  Maybe `stopFocusKeyPropagation`? Since it really is just stopping the propagation of the key along the focus chain? Might be too esoteric though.

Reply:
I went with `skipRemainingHandlers`, with some additional explanation.